src/main/java/de/embl/cba/mobie/n5/zarr/N5ZarrReader.java

```diff
@@ -164,7 +164,7 @@ public N5ZarrReader( final String basePath, final boolean mapN5DatasetAttributes
 	public N5ZarrReader( final String basePath, final GsonBuilder gsonBuilder) throws IOException
 	{
 
-		this(basePath, gsonBuilder, ".");
+		this(basePath, gsonBuilder, "/");
```

Comment:
Ok, I think this is the change that makes it work.
I think the proper way to fix this is to instead use the constructor that takes the dimensionSeparator as an argument
https://github.com/mobie/mobie-viewer-fiji/blob/5019b4f5d3e11517173db1f2bb03c70196eaa17f/src/main/java/de/embl/cba/mobie/n5/zarr/N5ZarrReader.java#L109
when it is called here
https://github.com/mobie/mobie-viewer-fiji/blob/5019b4f5d3e11517173db1f2bb03c70196eaa17f/src/main/java/de/embl/cba/mobie/n5/zarr/OMEZarrReader.java#L38